### PR TITLE
call onFetchData with last points per series

### PR DESCRIPTION
**Pull request by jasmineleeson**
allows calling application to be aware of the last data points for each series

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-212.surge.sh

File: src/components/DataProvider/index.js [hunk 1]
```diff
@@ -492,8 +492,16 @@ export default class DataProvider extends Component {
         [id]: { ...loaderConfig },
       };
     }
+
     this.setState(stateUpdates, () => {
-      this.props.onFetchData();
+      const lastPoints = {};
+      if (stateUpdates.contextSeries) {
```
Comment (evancharlton) [inline src/components/DataProvider/index.js]: Shouldn't you be iterating on `this.getSeriesObjects()` or something? `contextSeries` are series which will be drawn in the context chart (we should clean this up at some point ...)

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-212.surge.sh

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-212.surge.sh

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-212.surge.sh

**Pull request merged**